src/app/share/views/ShareView/ShareFileView.tsx

```diff
@@ -286,6 +286,7 @@ export default function ShareFileView(props: ShareViewProps): JSX.Element {
                         font-medium text-blue-60 active:bg-blue-20 active:bg-opacity-65"
             >
               <UilEye height="20" width="20" />
+              lala
```

Comment:
Remove "lala"

Reply:
I mark it as solved because I saw it in another [task](https://github.com/internxt/drive-web/pull/732) that has to be merged and it is already removed.